packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -2,9 +2,33 @@ import * as AdaptiveCard from 'adaptivecards';
 import MDTextInput from '@ciscospark/react-component-md-text-input';
 import MDToggleInput from '@ciscospark/react-component-md-toggle-input';
 import MDChoiceSetInput from '@ciscospark/react-component-md-choiceset-input';
+import {createMessageObject} from '@ciscospark/redux-module-activity';
 
 import {CARD_ATTACHMENT_TYPE, TEXT_INPUT_ELEMENT, TOGGLE_INPUT_ELEMENT, CHOICE_SET_INPUT_ELEMENT} from './constants';
 
+/**
+ * It used to handle markdown
+ * @param {string} text
+ * @param {object} result
+ * @returns {object}
+ */
+AdaptiveCard.AdaptiveCard.onProcessMarkdown = function processMarkdown(text, result = {}) {
+  const computedResult = {...result};
+
+  try {
+    const markedString = createMessageObject(text).content;
+
+    computedResult.outputHtml = markedString !== null ? markedString : text;
+    computedResult.didProcess = true;
+  }
+  catch (err) {
+    computedResult.didProcess = false;
+  }
+  Object.assign(result, computedResult);
```

Comment:
any reason we are not returning the `computedResult` instead of assigning it to the parameter and return the parameter?

Reply:
you are copying from the results and copying back to results. its looks a bit redundant to me.

Reply:
@akoushke, markdown is not working when I'm returning `computedResult` so that's reason again I assigned it to result

Reply:
It looks like we have to actually modify the `result` object based on the documentation: https://github.com/microsoft/AdaptiveCards/blob/master/source/nodejs/adaptivecards/README.md#supporting-markdown

In this case, let's go back to disabling the property change eslint rule, but just for this function. We just ask that you include the link to the docs and an explanation of why the eslint rule was disabled.

Reply:
disabled the property